content/en/blog/_posts/2024-08-15-moving-cgroup-v1-support-maintenance-mode-kubernetes-1-31.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: blog
+title: "Kubernetes 1.31: Moving cgroup v1 Support into Maintenance Mode in Kubernetes 1.31"
+date: 2024-08-15
+slug: kubernetes-1-31-moving-cgroup-v1-support-maintenance-mode
+author: Harshal Patil
+---
+
+As Kubernetes continues to evolve and adapt to the changing landscape of
+container orchestration, the community has decided to move cgroup v1 support
+into [maintenance mode](#what-does-maintenance-mode-mean) in v1.31.
+This shift aligns with the broader industry's move towards cgroup v2, offering
+improved functionality, scalability, and a more consistent interface. But before
+we dive into the consequences for Kubernetes, let's take a step back and
+understand what cgroups and their significance in Linux and Kubernetes are.
+
+## Understanding cgroups
+
+[Control groups](https://man7.org/linux/man-pages/man7/cgroups.7.html), or
+cgroups, are a Linux kernel feature that allows the allocation, prioritization,
+denial, and management of system resources (such as CPU, memory, disk I/O,
+and network bandwidth) among processes. This functionality is crucial for
+maintaining system performance and ensuring that no single process can
+monopolize system resources, which is especially important in multi-tenant
+environments.
+
+There are two versions of cgroups:
+[v1](https://docs.kernel.org/admin-guide/cgroup-v1/index.html) and
+[v2](https://docs.kernel.org/admin-guide/cgroup-v2.html). While cgroup v1
+provided sufficient capabilities for resource management, it had limitations
+that led to the development of cgroup v2. Cgroup v2 offers a more unified
+and consistent interface and better resource control features.
```

Comment:
I would go for 

"Cgroup v2 offers a more unified and consistent interface, on top of better resource control features." - avoids "and" repetition